shapeshift/web
# refactor: cosmos staking routers and implement new mockups

## Description

- Refactor every router from `Staking`, I was able to remove every memoryRouter used as they were adding too much complexity
- `Header` and `Steps` are now common to every Stake route inside `StakingView`
- Integrated/adjusted every view so it looks like the new mockups (See screenshots)

## Notice




## Pull Request Type


## Issue (if applicable)
closes #1266


## Risk



No much risk as it's behind a feature flag, but it touch the Cosmos staking part

## Testing

- Enable Cosmos feature flag
- Go to Cosmos asset page
- Click on Stake and then check every views



## Screenshots (if applicable)
![image](https://user-images.githubusercontent.com/14963751/159995276-d5d0791d-4204-4539-8020-0994c13bceda.png)
![image](https://user-images.githubusercontent.com/14963751/159995325-105a3f5c-ef44-49af-b356-bb61e2ce138c.png)
![image](https://user-images.githubusercontent.com/14963751/159995358-5123ef8b-1242-47f1-a564-0aa9c70ea3ed.png)
![image](https://user-images.githubusercontent.com/14963751/159995401-3156d8d8-2f35-4e86-9e8e-6aba03f6e762.png)
![image](https://user-images.githubusercontent.com/14963751/159995436-c02d3140-6da0-4ec1-a239-4d1e15cb7b0e.png)
![image](https://user-images.githubusercontent.com/14963751/159995478-7b0f6a7b-5238-462d-8659-f53c145059b6.png)
![image](https://user-images.githubusercontent.com/14963751/159995504-433670c4-7c30-4797-bc42-47671f356ad6.png)
![image](https://user-images.githubusercontent.com/14963751/159995534-56317138-a80e-450c-a465-6e1820e10c36.png)
![image](https://user-images.githubusercontent.com/14963751/159995568-9762d887-7fd2-4c62-85aa-32676cac2677.png)

### Video link
https://www.awesomescreenshot.com/video/8053229?key=81e76c97917b316f5e5c0b7b58d6b296
